src/bin/maker-cli.rs

```diff
@@ -37,14 +37,8 @@ enum Commands {
     ListUtxoContract,
     /// Lists fidelity bond utxos.
     ListUtxoFidelity,
-    /// Get total wallet balance, excluding Fidelity bonds.
+    /// Get total wallet balance.
```

Comment:
The doc can be enhanced a bit like below.
```
/// Get total wallet balances of different categories.
/// regular: all single signature regular wallet coins.
/// swap: all 2of2 multisig coins received in swaps.
/// contract: All live contract transaction balance locked in timelocks. If you see value in this field, you have unfinished or malfinished swaps. You can claim them back with recover command.
/// fidleity: all coins locked in fidelity bonds.
```
As now its the only balance doc, we need to explain clearly what each balance means.